base/broadcast.jl

```diff
@@ -110,6 +129,9 @@ const bitcache_size = 64 * bitcache_chunks # do not change this
 dumpbitcache(Bc::Vector{UInt64}, bind::Int, C::Vector{Bool}) =
     Base.copy_to_bitarray_chunks!(Bc, ((bind - 1) << 6) + 1, C, 1, min(bitcache_size, (length(Bc)-bind+1) << 6))
 
+@inline _broadcast_getindex(A, I) = A
```

Comment:
The underscore seems unnecessary here.  We don't add underscores before the other function that are mostly internal, like `broadcast_indices`.